qtawesome/iconic_font.py

```diff
@@ -197,14 +202,32 @@ def hook(obj):
         with open(os.path.join(directory, charmap_filename), 'r') as codes:
             self.charmap[prefix] = json.load(codes, object_hook=hook)
 
-        id_ = QFontDatabase.addApplicationFont(os.path.join(directory, ttf_filename))
+        ttf_hash_code = None
+        if ttf_filename == 'fontawesome-webfont.ttf':
+            ttf_hash_code = 'a3de2170e4e9df77161ea5d3f31b2668'
+        elif ttf_filename == 'elusiveicons-webfont.ttf':
+            ttf_hash_code = '207966b04c032d5b873fd595a211582e'
+        if ttf_hash_code != None:
```

Comment:
If we use a dictionary for hashes then this has to be

```python
ttf_hash = md5_hashes.get(ttf_filename, None)
if ttf_hash is not None:
    ...
```

And yes, please use `ttf_hash` instead of `ttf_hash_code` for this variable.